src/app/_components/gift-card.tsx

```diff
@@ -20,27 +21,33 @@ const GiftCard: React.FC<GiftCardProps> = ({
   restaurant,
 }) => {
   return (
-    <Link href={`/search/${name}`}>
-      <div className="relative flex h-72 w-56 cursor-pointer flex-col items-start rounded-sm bg-white p-0 shadow-sm">
-        <div className="absolute h-[60.69%] w-full rounded-t-sm bg-[url('/img/flipa-box.svg')]"></div>
-        <div className="left-5.38% right-5.5% absolute top-[65.52%] flex w-full flex-row items-center justify-between p-0">
-          <h3 className="font-poppins text-lg font-medium leading-[108%] text-black">
-            {name}
-          </h3>
-          <div className=" bottom-4 right-0 flex items-center">
-            <span className="bg-gradient-to-r from-blue-700 to-blue-300 bg-clip-text font-poppins text-sm font-normal text-transparent">
-              {calification}
-            </span>
-            <img src="/card/star.svg" className="h-3 w-3 text-blue-500" />
+    <Link href={`/search/${name}`} className="w-full">
+      <div className="flex aspect-square flex-col gap-2 overflow-hidden rounded-lg shadow-lg">
+        <div className="h-full w-full">
+          <div className="relative aspect-square h-[60%] w-full ">
+            <div className="absolute right-2 top-2 z-10 h-3 w-3.5 bg-[url('/card/favorite.svg')] bg-cover bg-center bg-no-repeat"></div>
+            <div className="absolute left-0 top-0 h-full w-full rounded-t-lg bg-[url('/img/flipa-box.svg')] bg-cover bg-center"></div>
+          </div>
+          <div className="p-2">
```

Comment:
nitpick: let's try to add some blank lines when we have really long code blocks. These two divs are logically independent, so we can add a blank line between them and that way improve code readability